inc/config.class.php

```diff
@@ -81,37 +81,37 @@ static function install(Migration $mig) {
+                     `is_ticketsolutiontype_mandatory` tinyint NOT NULL default '0',
+                     `is_ticketsolution_mandatory` tinyint NOT NULL default '0',
+                     `is_ticketcategory_mandatory` tinyint NOT NULL default '0',
+                     `is_ticketcategory_mandatory_on_assign` tinyint NOT NULL default '0',
+                     `is_tickettaskcategory_mandatory` tinyint NOT NULL default '0',
+                     `is_tickettech_mandatory` tinyint NOT NULL default '0',
+                     `is_tickettechgroup_mandatory` tinyint NOT NULL default '0',
+                     `is_ticketrealtime_mandatory` tinyint NOT NULL default '0',
+                     `is_ticketlocation_mandatory` tinyint NOT NULL default '0',
+                     `is_requester_mandatory` tinyint NOT NULL default '0',
+                     `is_ticketdate_locked` tinyint NOT NULL default '0',
+                     `use_assign_user_group` tinyint NOT NULL default '0',
+                     `use_assign_user_group_update` tinyint NOT NULL default '0',
+                     `ticketsolved_updatetech` tinyint NOT NULL default '0',
                      `tickets_id_format` VARCHAR(15) NULL,
                      `changes_id_format` VARCHAR(15) NULL,
-                     `is_problemsolutiontype_mandatory` tinyint(1) NOT NULL default '0',
-                     `remove_from_ocs` tinyint(1) NOT NULL default '0',
-                     `add_notif` tinyint(1) NOT NULL default '0',
-                     `use_lock` tinyint(1) NOT NULL default '0',
+                     `is_problemsolutiontype_mandatory` tinyint NOT NULL default '0',
+                     `remove_from_ocs` tinyint NOT NULL default '0',
+                     `add_notif` tinyint NOT NULL default '0',
+                     `use_lock` tinyint NOT NULL default '0',
                      `single_tech_mode` int(11) NOT NULL default '0',
-                     `myasset` tinyint(1) NOT NULL default '0',
-                     `groupasset` tinyint(1) NOT NULL default '0',
-                     `clone` tinyint(1) NOT NULL default '0',
-                     `is_tickettasktodo` tinyint(1) NOT NULL default '0',
+                     `myasset` tinyint NOT NULL default '0',
+                     `groupasset` tinyint NOT NULL default '0',
+                     `clone` tinyint NOT NULL default '0',
+                     `is_tickettasktodo` tinyint NOT NULL default '0',
                      `date_mod` datetime default NULL,
                      `comment` text,
                      PRIMARY KEY  (`id`)
-                   ) ENGINE=InnoDB  DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci";
+                   ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci ROW_FORMAT=DYNAMIC;";
          $DB->queryOrDie($query, __('Error in creating glpi_plugin_behaviors_configs', 'behaviors').
                                  "<br>".$DB->error());
 
```

Comment:
change but not like you

Reply:
Hello,
Can anyone tell me how to make an additional field mandatory at ticket close please?
Regards.